src/kv/source.rs

```diff
@@ -374,6 +374,12 @@ mod std_support {
     }
 }
 
+// REVIEW: I'm not a 100% convinced that AsmAp and AsList are pulling their
```

Comment:
These also seem fair to remove and consider adding later, or adding in a utilities crate :+1: